recipes/r-fossil/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = '0.4.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fossil
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fossil_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fossil/fossil_{{ version }}.tar.gz
+  sha256: 37c082fa15ebae89db99d6071b2bb2cad6a97a0405e9b4ef77f62a8f6ad274c1
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
```

Comment:
Please remove the comments except of the cran meta data or rerun with newest conda-build skeleton cran and the `--no-comments` option. thx.